src/tabs/__tests__/__snapshots__/stateful-tabs.test.js.snap

```diff
@@ -1,5 +1,101 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`StatefulTabs basic render one tab: Stateful tabs render correctly 1`] = `
```

Comment:
can you add a visual regression test instead of the snapshot, please? this is an example for it https://github.com/uber-web/baseui/blob/master/src/button-group/__tests__/button-group-radio.scenario.js